### PR TITLE
Added custom system tests repo and branch input option

### LINKED ISSUE #93: Add option to pass in adoptium aqa-systemtest repo and branch
Currently we are using the default aqa-systemtest master branch for jck and system tests. It would be nice to add adoptium aqa-systemtest repo and branch as option. So run-aqa action can also be used as the part of aqa-systemtest  developer's PR tests.

### PR DESCRIPTION
**Pull request by Vindiya**
fixes #93

### CODE REVIEW

File: src/runaqa.ts [hunk 1]
```diff
@@ -57,6 +58,9 @@ export async function runaqaTest(
   await exec.exec(
     `${sevenzexe} e ${dependents} -o${process.env.GITHUB_WORKSPACE}/aqa-tests/TKG/lib`
   )
+  if(buildList.includes('system') || buildList.includes('openjdk')) {
```
Comment (sophia-guo) [inline src/runaqa.ts:61]: openjdk tests is a different category from system. So we can just move this function call getAqaSystemTestsRepo to the following block https://github.com/adoptium/run-aqa/pull/102/files#diff-22673e27ee79038a5688a185a2cd7775b8d2a63bfb6c4b449d1d5195cc272faaR65

`if (buildList.includes('system')) {`

File: src/runaqa.ts [hunk 2]
```diff
@@ -229,6 +233,17 @@ async function getAqaTestsRepo(aqatestsRepo: string): Promise<void> {
   process.chdir('aqa-tests')
 }
 
+async function getAqaSystemTestsRepo(aqasystemtestsRepo: string): Promise<void> {
+  let repoBranch = ['adoptium/aqa-systemtests', 'master']
+  if (aqasystemtestsRepo.length !== 0) {
+    repoBranch = parseRepoBranch(aqasystemtestsRepo)
+  }
+  await exec.exec(
```
Comment (sophia-guo) [inline src/runaqa.ts:241]: For Aqasystemtest the only thing we need is to export the repo and branch.

```
process.env.ADOPTOPENJDK_SYSTEMTEST_REPO = repoBranch[0]
process.env.ADOPTOPENJDK_SYSTEMTEST_BRANCH = repoBranch[1]
```

**Comment by Vindiya** [comment]
#93

**Comment by smlambert** [comment]
Waiting for checks to complete...

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**